typescript/src/schema/proto.ts

```diff
@@ -491,6 +563,11 @@ function augmentMethod(
   parameters: AugmentMethodParameters,
   method: MethodDescriptorProto
 ) {
+  // whether a service is allowed to use Int32Value and UInt32Value wrappers - generally this is only BigQuery
+  // this is used to determine factors about pagination fields and to allow users to pass a "number" instead of
+  // having to convert to a protobuf wrapper type to determine page size
+  const wrappersAllowed =
+    ENABLE_WRAPPER_TYPES_FOR_PAGE_SIZE[parameters.service.packageName];
```

Comment:
I think if you don't want pubsub to change, you'd make this:
```ENABLE_WRAPPER_TYPES_FOR_PAGE_SIZE[parameters.service.packageName] ? true : undefined```
This would preserve current behavior but it would also make it less clear in the future. However, do you envision this list being actually a full dict of opt-in or opt-outs? I could also see `ENABLE_WRAPPER_TYPES_FOR_PAGE_SIZE` just being an array where we assume opt-in. I'm fine either way, perhaps it's best to talk to pubsub since it seems they're the only other ones potentially affected by this change (i.e., do they want to explicitly opt-out or just continue functionality?)

Reply:
@feywind thoughts? I'll set up time to have a quick chat to see if this would help or hurt things on your end

Reply:
RE whether it should be opt-in, I think it should be opt-ins

Reply:
Yeah, let's maybe chat about it, but it sounds like it's probably not going to be an issue, as long as it doesn't require a major bump in Pub/Sub.